tests/sentry/incidents/endpoints/test_project_alert_rule_details.py

```diff
@@ -424,15 +389,124 @@ def test_no_owner(self):
         assert resp.data == serialize(alert_rule, self.user)
         assert resp.data["owner"] is None
 
+    @responses.activate
+    def test_success_response_from_sentry_app(self):
+        responses.add(
+            method=responses.POST,
+            url="https://example.com/sentry/alert-rule",
+            status=202,
+        )
+
+        sentry_app = self.create_sentry_app(
+            name="foo",
+            organization=self.organization,
+            schema={
+                "elements": [
+                    self.create_alert_rule_action_schema(),
+                ]
+            },
+        )
+        install = self.create_sentry_app_installation(
+            slug="foo", organization=self.organization, user=self.user
+        )
+
+        sentry_app_settings = [
+            {"name": "title", "value": "test title"},
+            {"name": "description", "value": "test description"},
+        ]
+
+        test_params = self.valid_params.copy()
+        test_params["triggers"] = [
+            {
+                "actions": [
+                    {
+                        "type": "sentry_app",
+                        "targetType": "sentry_app",
+                        "targetIdentifier": sentry_app.id,
+                        "hasSchemaFormConfig": True,
+                        "sentryAppId": sentry_app.id,
+                        "sentryAppInstallationUuid": install.uuid,
+                        "settings": sentry_app_settings,
+                    }
+                ],
+                "alertThreshold": 300,
+                "label": "critical",
+            }
+        ]
+
+        with self.feature(["organizations:incidents", "organizations:performance-view"]):
+            self.get_valid_response(
+                self.organization.slug,
+                self.project.slug,
+                self.alert_rule.id,
+                status_code=200,
+                **test_params,
+            )
+
+    @responses.activate
+    def test_error_response_from_sentry_app(self):
```

Comment:
This tests that the error message from Sentry Apps will be surfaced for metric alerts (PUT endpoint)